client/restclient_test.go

```diff
@@ -49,7 +50,7 @@ func Test_apiUpdate(t *testing.T) {
 
 			// use payload matching /v2/imagefile/{ref}/_complete endpoint. This
 			// is an arbitrary test of apiUpdate not specific to this endpoint.
-			res, err := c.apiUpdate(ctx, tt.url, UploadImageCompleteRequest{})
+			res, err := c.apiUpdate(ctx, strings.TrimPrefix(tt.url, "/"), UploadImageCompleteRequest{})
```

Comment:
Might be better to update the `tt.url` values rather than modifying in the test?